pytorch_lightning/utilities/apply_func.py

```diff
@@ -27,11 +30,18 @@
     Batch = type(None)
 
 
+CONVERSION_DTYPES = [
+    (bool, torch.bool),
+    (int, torch.int),
+    (float, torch.float),
+    (np.ndarray, None),
```

Comment:
alternatively you could define a callable (torch.from_numpy) here instead of specifically checking for ndarray below